### PR TITLE
Move pkg/kubectl/util/printers to staging

### PR DESCRIPTION
**Pull request by damemi**
* Moves pkg/kubectl/util/printers to staging
* Publishes this code to the new kubectl repository: https://github.com/kubernetes/kubectl/pkg/util/printers.
* Code will be available for re-use after publishing.

/kind cleanup
/sig cli
/area kubectl

```release-note
NONE
```

### CODE REVIEW

**Comment by damemi** [comment]
cc @seans3

**Review by seans3 — COMMENTED** [review]
Looks good. We can rebase after the templates/term/interrupt PR goes in.

File: staging/src/k8s.io/kubectl/go.mod [hunk 1]
```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
 	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
+	github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
```
Comment (apelisse) [inline staging/src/k8s.io/kubectl/go.mod]: That looks new and weird, what's going on?

Comment (apelisse) [reply staging/src/k8s.io/kubectl/go.mod]: OK Actually I think it makes sense to add that dependency.

**Comment by apelisse** [comment]
for the go mod/vendoring part:
/approve

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**